mlir/include/mlir/Dialect/LLVMIR/ROCDLOps.td

```diff
@@ -165,7 +165,7 @@ def ROCDL_BallotOp :
   let summary = "Vote across thread group";
 
   let description = [{
-      Ballot provides a bit mask containing the 1-bit predicate value from each lane. 
+      Ballot provides a bit mask containing the 1-bit predicate value from each lane.
```

Comment:
Unrelated whitespace change

Reply:
I can add the trailing white space back, but git is warning me against this:
```
warning: 1 line adds whitespace errors.
```
My preference would be to remove the trailing whitespace, but I can remove it if you think it makes the PR cleaner.

Reply:
Keep it out of this PR, can just precommit unrelated whitespace changes

Reply:
Done